site/content/docs/4.3/forms/checks.md

```diff
@@ -209,3 +209,54 @@ Omit the wrapping `.form-check` for checkboxes and radios that have no label tex
   <input class="form-check-input" type="radio" name="radioNoLabel" id="radioNoLabel1" value="" aria-label="...">
 </div>
 {{< /example >}}
+
+## Toggle buttons
```

Comment:
We call these toggle buttons here, but the language used on the radio/checkbox base classes has been `.form-check`. Maybe we keep the heading here as `Toggle buttons`, but swap the class for `.btn-check`? Caught myself getting confused too because of `.dropdown-toggle` being the class you interact wiht in dropdowns.